tern/analyze/common.py

```diff
@@ -563,9 +563,54 @@ def get_git_sha(path_to_toplevel):
     return sha_info
 
 
+def get_git_url(dockerfile_path):
+    '''Given a dockerfile_path, return url of git project which contains
+    the dockerfile in a form of list.'''
+    # get the path of the folder containing the dockerfile
+    dockerfile_folder_path = os.path.dirname(dockerfile_path)
```

Comment:
I use the dockerfile_folder_path as the cwd which assums that the dockerfile is under a git project.